canonical/ubuntu.com
# Update new Dell logo instances

## Done

- Update new Dell logo throughout all pages
- Apply djlint formatting to modified files (source of big diff)
- List of copy docs: https://docs.google.com/document/d/1XWYk5IWx9US6ympIMNveRlgJOH_cyNCB1eQFHBsr4tA/edit

## QA
- Go to these websites and check that the Dell logos are updated accordingly as per copy doc
  - https://ubuntu-com-14081.demos.haus/
  - https://ubuntu-com-14081.demos.haus/desktop
  - https://ubuntu-com-14081.demos.haus/desktop/developers
  - https://ubuntu-com-14081.demos.haus/desktop/organisations
  - https://ubuntu-com-14081.demos.haus/kubernetes/resources
  - https://ubuntu-com-14081.demos.haus/server
  - https://ubuntu-com-14081.demos.haus/certified
  - https://ubuntu-com-14081.demos.haus/ai/roadshow
  - https://ubuntu-com-14081.demos.haus/gov
  - https://ubuntu-com-14081.demos.haus/appliance/hardware
  - https://ubuntu-com-14081.demos.haus/industrial
  - https://ubuntu-com-14081.demos.haus/edge-computing
  - https://ubuntu-com-14081.demos.haus/download/desktop
  - https://ubuntu-com-14081.demos.haus/dell
  - https://ubuntu-com-14081.demos.haus/internet-of-things/appstore
  - https://ubuntu-com-14081.demos.haus/ai/data-science

## Issue / Card

Fixes [WD-13284](https://warthogs.atlassian.net/browse/WD-13284)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-13284]: https://warthogs.atlassian.net/browse/WD-13284?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
